{{cookiecutter.repo_name}}/CONTRIBUTING.rst

```diff
@@ -91,6 +91,9 @@ Now you can make your changes locally.
     # to run coverage test only, run:
     $ pipenv run coverage
 
+    # to run the pytest tests, run:
+    $ pipenv run pytest
+
```

Comment:
I think it would be a good idea to add some instructions on bumping versions in the PR somewhere around here too.

Reply:
I agree, doing that now.